src/scripts/appclose.ts

```diff
@@ -0,0 +1,11 @@
+((require as unknown) as typeof define)(["playbackManager"], function (playbackManager) {
+    window["AppCloseHelper"] = {
+        onClosing: function (): void {
+            // Prevent backwards navigation from stopping video
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Add a simple "return" instead of disabling the rule.

Reply:
I thought about completely disabling this rule it doesn't make a lot of sense if you know what you are doing

Reply:
At some point, you won't be the only contributor to this. We can't assume that people know what they're doing. Linting rules are strict for a reason.

Reply:
Yeah I'll try adding a return

Reply:
I just find it really common to have to use empty functions and there sadly no automatic fix for it